flutter-for-android.md

```diff
@@ -0,0 +1,1864 @@
+---
+layout: page
+title: Flutter for Android Developers
+permalink: /flutter-for-android/
+---
+This document is meant for Android developers looking to transfer their 
+existing Android knowledge to build mobile apps in Flutter. If you understand 
+the fundamentals of the Android framework then you can use this document as a 
+jump start to Flutter development.
+
```

Comment:
Let's add a disclaimer here:

Your Android knowledge and skill set are highly valuable when building with Flutter, because Flutter relies on the mobile operating system for numerous capabilities and configurations. Flutter is a new way to build UIs for mobile, but it has a plugin system to communicate with Android (and iOS) for non-UI tasks. If you're an expert with Android, you don't have to relearn everything to use Flutter.